easybuild/easyconfigs/r/R/R-4.0.5-foss-2020b.eb

```diff
@@ -0,0 +1,2852 @@
+        'checksums': ['ba3a1ab6ffe92f592c9f2bb1d4070f1fb1019325226dcb4863cf725eb59e9b2d'],
+    }),
+    ('nlsem', '0.8', {
+        'checksums': ['495a5d07aa5f59efdcd43acf429ae842453abd6c0720a80e2102d663fa997c60'],
+    }),
+    ('mitools', '2.4', {
+        'checksums': ['f204f3774e29d79810f579f128de892539518f2cbe6ed237e08c8e7283155d30'],
+    }),
+    ('survey', '4.0', {
+        'checksums': ['b053f40f4cfa90507ca524f72d3b3a4b4869def52f11f907a14f1c6d90063de1'],
+    }),
+    ('tableone', '0.12.0', {
+        'checksums': ['6a5cc16f7d2303c8f42b8adcad0fc41e1ba74f24ada4e7ad3a16effb63d3575e'],
+    }),
+    ('jstable', '1.0.2', {
+        'checksums': ['0d7ffa8ecd1b2df05cf1256406152d211eabfa660bf1cc13bada1d75dfdd6d13'],
+    }),
+    ('RCAL', '2.0', {
+        'checksums': ['10f5f938a8322d8737159e1e49ce9d12419a5130699b8a19c6ca53d6508da8cc'],
+    }),
+    ('stargazer', '5.2.2', {
+        'checksums': ['70eb4a13a6ac1bfb35af07cb8a63d501ad38dfd9817fc3fba6724260b23932de'],
+    }),
+    ('sensemakr', '0.1.3', {
+        'checksums': ['2eccda4ac3752266779d9c8ae87154c9fbaf0f73e0a768692a836a29ceaeffdd'],
+    }),
+    ('CompQuadForm', '1.4.3', {
+        'checksums': ['042fc56c800dd8f5f47a017e2efa832caf74f0602824abf7099898d9708660c4'],
+    }),
+    ('nonnest2', '0.5-5', {
+        'checksums': ['027f510e322122fc75c936251a95ddd392f96047ac86e0fae6cf8f883ac7aab5'],
+    }),
+    ('blavaan', '0.3-15', {
+        'checksums': ['f73ead024bc3b65bdb0c5e5cd5458845158914eb579c07be2fd697a3573ebe6f'],
+    }),
+    ('metafor', '2.4-0', {
+        'checksums': ['b64a678b762e91f1e0a6360b15e79fe19159b243c9f40ad9cc0be833bb4ba9ac'],
+    }),
+    ('fmri', '1.9.6', {
+        'checksums': ['7614290d880667512744d3450480a670cc38abdb270f3f776ac9a17a793f07f2'],
+    }),
+    ('AnalyzeFMRI', '1.1-23', {
+        'checksums': ['4237ca4ffe812e614349cdffa2668de4eda5697d73ea208df0c60808daa29444'],
+    }),
+    ('linkcomm', '1.0-14', {
+        'checksums': ['36f1557c65d862fc87635eedfad77f18a5deb66da00895e50e2d5eac0f23b597'],
+    }),
+    ('rnetcarto', '0.2.4', {
+        'checksums': ['266702330250e9fbeb8616d86edf1d50d63084a0731d17e84a04dc6faacf653a'],
+    }),
+    ('DEoptim', '2.2-5', {
+        'checksums': ['ae12dedcd4a43994e811e7285f8c12bfdb688e7c99d65515cf7e8cb6db13955a'],
+    }),
+    ('optextras', '2019-12.4', {
+        'checksums': ['59006383860826be502ea8757e39ed94338f04d246c4fc398a088e004d8b13eb'],
+    }),
+    ('setRNG', '2013.9-1', {
+        'checksums': ['1a1a399682a06a5fea3934985ebb1334005676c6a2a22d06f3c91c3923432908'],
+    }),
+    ('Rvmmin', '2018-4.17', {
+        'checksums': ['d53ba7ab06596a47990caf101a50935b2b34402f9dd8414f098a873026ff1f56'],
+    }),
+    ('Rcgmin', '2013-2.21', {
+        'checksums': ['a824a09c32d7565a3e30607c71333506d5b7197478fbe8b43f8a77dad6c12f0a'],
+    }),
+    ('optimr', '2019-12.16', {
+        'checksums': ['73b1ed560ffd74599517e8baa4c5b293aa062e9c8d50219a3a24b63e72fa7c00'],
+    }),
+    ('DMCfun', '1.3.0', {
```

Comment:
@robert-mijakovic Please also include the patch that was added in #12583 (same in both easyconfigs)

Reply:
Done!